src/components/Game/Game.js

```diff
@@ -86,26 +87,35 @@ class GameView extends Component {
         });
     };
 
-    onSubmit = () => {
+    submit = () => {
         const { lettersInSlots } = this.state;
         const { dispatch, history } = this.props;
         dispatch(submitAnswer(this.joinLetters(lettersInSlots)));
         history.replace('/result');
     };
 
+    onSubmit = () => {
+        this.submit();
+    };
+
     onCancel = () => {
         this.setState({
             isSubmitVisible: false,
         });
     };
 
+    timeIsOver = () => {
+        this.submit();
```

Comment:
should we submit an answer if it's shorter than 7 letters?
this is design decision
if the answer is yes then it may lead to the below UI (is it okay?):
![React App](https://user-images.githubusercontent.com/2458313/65419323-b1724b00-ddfe-11e9-8c79-f5bccf50cb37.png)

Reply:
good question! I thought it was natural that we did but now I think the opposite since we require a player to form a 7-letter word. I wouldn't like it to cause any confusion regarding the rules.

but when it comes to technicalities I think I would still submit a shorter answer (meaning dispatch an action, probably with an empty string) because we need to identify 3 different situations:
1. a game was on and a player submitted a 7-letter answer
2. a game was on and a player submitted a shorter answer or no answer (slots were empty on submit)
3. there was no game so we need to redirect

what do u think?

Reply:
> I think it would be better not to display 3-letter words on Result page and treat this case the same as "a game was on and a player didn't submit answer".

yes, I wasn't very clear maybe but I meant this exactly.